python/creds_template.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
As above - this file should not be part of your pull request (and shouldn't be part of any repo really - as a general security practice keys should be part of a configuration file that is not checked in anywhere).

Reply:
Same...